.changeset/old-actors-learn.md

```diff
@@ -0,0 +1,11 @@
+---
+'create-astro': minor
+---
+
+Rework the experience of creating a new Astro project.
+
+- Update the list of templates to include Starlight and remove the minimal template.
+- Removed the TypeScript question. Astro is TypeScript-only, so this question was often misleading. The "Strict" preset is now the default, but it can still be changed manually in `tsconfig.json`. `astro check` is no longer automatically added to the build script.
```

Comment:
```suggestion
- Removes the TypeScript question. Astro is TypeScript-only, so this question was often misleading. The "Strict" preset is now the default, but it can still be changed manually in `tsconfig.json`. 
- `astro check` is no longer automatically added to the build script.
```
I know they're both TypeScript related, but it seemed like there were different, separate reasons for removing `astro check` so this probably deserves its own line? And, we could put an explanation of why if you wanted!